addon/mixins/gatekeeper.js

```diff
@@ -187,6 +185,21 @@ export default Mixin.create({
     return result;
   }),
 
+  /**
+   * Return the full url to the Portal's REST API
+   */
+  portalRestUrl: computed('isAuthenticated', function () {
+    let result;
+    if (this.get('isAuthenticated')) {
+      result = getPortalRestUrl(this.get('portal'));
```

Comment:
I think we need to add `'portal'` to the CP's dependencies.

Reply:
added